### PR TITLE
Added test relating to issue #4517

### PR DESCRIPTION
**Pull request by nrydanov**
Closes: #4517

### CODE REVIEW

**Review by niksv — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy [hunk 1]
```diff
@@ -158,12 +158,12 @@ class MirrorEndpointsSpec extends HealthCheckSpecification {
         and: "Original flow rule counter is not increased"
         flowRule.packetCount == findFlowRule(getFlowRules(swPair.src.dpId), mirrorDirection).packetCount
 
-        //https://github.com/telstra/open-kilda/issues/4517
-//        and: "System collects stat for mirror cookie in otsdb"
-//        Wrappers.wait(statsRouterRequestInterval) {
-//            def tags = [flowid: flow.flowId, cookie: mirrorRule.cookie]
-//            assert otsdb.query(genTrafficTime, metricPrefix + "flow.raw.bytes", tags).dps.size() > 0
-//        }
+        // https://github.com/telstra/open-kilda/issues/4517
```
Comment (dpoltavets) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy:161]: Why is this comment here?

**Review by artHalMirantis — COMMENTED** [review]
can be closed\merged once the comments will be resolved

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy (hunk 1 above)
Comment (nrydanov) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy]: This comment was there before I started an issue. I thought it is important for QA

**Comment by nrydanov** [comment]
> can be closed\merged once the comments will be resolved

Fixed. You may approve then.

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy (hunk 1 above)
Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy]: It explained why certain part of the test was commented and also marked part of the test that can verify certain issue. As soon as issue is fixed the comment can be removed

Comment (nrydanov) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy]: Thanks for the explanation

**Review by dpoltavets — APPROVED** [review]

**Review by vkhalin — APPROVED** [review]

**Pull request merged**